src/main/java/org/toradocu/extractor/BlockTag.java

```diff
@@ -11,7 +10,7 @@
  * specification (available after the translation of the comment). A specification is the
  * translation of the comment into a Java boolean condition.
  */
-public abstract class BlockTag<S extends Specification> {
+public abstract class BlockTag {
```

Comment:
Those classes represent Javadoc comments. I think comments are rather different than specifications. One example is the `@return` block tag: to a single comment correspond at list two Randoop PostSpecification(s). Also in this way we decouple the Javadoc information extracted from the Java code from its usage.